### PR TITLE
Upgrade to redispy 4.1.3 (min python 3.6)

### PR DESCRIPTION
**Pull request by MeirShpilraien**
Moved to redispy 4.1.3 and add tls-passphrase option

- [x] Require this fix on redispy : https://github.com/redis/redis-py/pull/1926

**The PR changes minimum python supported version to 3.6**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -24,9 +24,9 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "^2.7,<2.8 || >= 3.5.0"
+python = ">= 3.6.0"
 distro = "^1.5.0"
-redis = "^3.5.3"
+redis = "^4.1.2"
 redis-py-cluster = "*"
```
Comment (chayim) [inline pyproject.toml:30]: I think you want to remove redis-py-cluster here.

File: pyproject.toml [hunk 2]
```diff
@@ -24,9 +24,9 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "^2.7,<2.8 || >= 3.5.0"
+python = ">= 3.6.0"
```
Comment (chayim) [inline pyproject.toml:27]: python = ">= 3.6.0 || < 4.0.0"

**Review by chayim — COMMENTED** [review]
Small suggestions. Great PR - surgical.

File: pyproject.toml (hunk 1 above)
Comment (MeirShpilraien) [reply pyproject.toml]: Done

File: pyproject.toml (hunk 2 above)
Comment (MeirShpilraien) [reply pyproject.toml]: Done

**Review by chayim — APPROVED** [review]

**Pull request merged**